### PR TITLE
chore: add itests for pandas

### PR DESCRIPTION
**Pull request by bojiang**
## Checklist:

- [ ] My code follows the bentoml code style, both `./dev/format.sh` and
  `./dev/lint.sh` script have passed
  ([instructions](https://github.com/bentoml/BentoML/blob/master/DEVELOPMENT.md#style-check-and-auto-formatting-your-code)).
- [ ] My change reduces project test coverage and requires unit tests to be added
- [ ] I have added unit tests covering my code change
- [ ] My change requires a change to the documentation
- [ ] I have updated the documentation accordingly

### CODE REVIEW

File: bentoml/_internal/io_descriptors/pandas.py [hunk 1]
```diff
@@ -108,7 +108,7 @@ def infer(input_df: pd.DataFrame) -> pd.DataFrame:...
     def __init__(
         self,
         orient: Literal[
-            "dict", "list", "series", "split", "records", "index"
+            'split', 'records', 'index', 'columns', 'values', 'table'
```
Comment (bojiang) [inline bentoml/_internal/io_descriptors/pandas.py:111]: @aarnphm actual valid orients for DataFrames

File: bentoml/_internal/io_descriptors/pandas.py [hunk 2]
```diff
@@ -256,9 +264,7 @@ class PandasSeries(PandasDataFrame):
 
     def __init__(
         self,
-        orient: Literal[
-            "dict", "list", "series", "split", "records", "index"
-        ] = "records",
+        orient: Literal['split', 'records', 'index', 'table'] = "records",
```
Comment (bojiang) [inline bentoml/_internal/io_descriptors/pandas.py:267]: valid orients for Series

File: bentoml/_internal/io_descriptors/pandas.py (hunk 1 above)
Comment (aarnphm) [reply bentoml/_internal/io_descriptors/pandas.py:111]: you can't input other type of orient since it is a Literalt type right?

Comment (bojiang) [reply bentoml/_internal/io_descriptors/pandas.py:111]: yeah. my point is, 'dict' & 'list' are not valid orients

**Pull request merged**